Rideau/SwiftUI/SwiftUISuppports.swift

```diff
@@ -0,0 +1,163 @@
+import UIKit
+
+enum SwiftUISupports {
+  final class RideauHostingController: UIViewController {
+
+    // MARK: - Properties
+
+    var onWillDismiss: () -> Void = {}
```

Comment:
Not called?

Reply:
hm, yes, that's called.
but anymore it's not meaningful.
I'll delete it.